runtime/permissions/prompter.rs

```diff
@@ -117,7 +124,10 @@ fn clear_stdin(
     loop {
       let r = libc::tcflush(STDIN_FD, libc::TCIFLUSH);
       if r != 0 {
-        bail!("clear_stdin failed (tcflush)");
+        return Err(
+          deno_core::error::generic_error("clear_stdin failed (tcflush)")
+            .into(),
```

Comment:
Might as well just make this an io error then have this return `std::io::Error`?